common/src/main/webapp/theme/semantic.less

```diff
@@ -130,9 +130,9 @@
 & {
   @import "fomantic-ui-less/definitions/modules/nag";
 }
-& {
-  @import "fomantic-ui-less/definitions/modules/popup";
-}
+// & {
```

Comment:
Commented code

Reply:
Maybe the idea is to leave it like this?

Reply:
Yes I'm going to leave it there, we could remove a few more and reduce css size